basic-algorithm-scripting/boo-who.js

```diff
@@ -0,0 +1,9 @@
+function booWho(bool) {
+  if (bool === true || bool === false) {
+    return true;
+  } else {
+    return false;
```

Comment:
With that all this code can be removed... returning only the expression typeof bool is boolean because this expression already returns true or false depending if bool is boolean or not. So you dont need all these ifs and return true/return false